doc/devel/coding_guide.rst

```diff
@@ -47,6 +47,18 @@ PR Review guidelines
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
+* Tag the PR with appropriate labels.
+
+* Set the milestone to
+
+  * next point-release for new features.
```

Comment:
Can there be a bit more explanatory text for what this is, even if it's 3.X for feature release vs 3.X.X for bugfix release